.circleci/gradle.properties

```diff
@@ -1,4 +1,4 @@
-org.gradle.jvmargs=-Xmx7200M -Dkotlin.daemon.jvm.options\="-Xmx7200M"
-robolectricHeapSize=7200m
+org.gradle.jvmargs=-Xmx3686M -Dkotlin.daemon.jvm.options\="-Xmx3686M"
```

Comment:
I think the container probably needs a little more mem for itself. Maybe try 3072m?